catalog/datasets/muziekschatten-onderwerpen.jsonld

```diff
@@ -0,0 +1,34 @@
+{
+  "@context": "https://schema.org/docs/jsonldcontext.jsonld",
+  "@id": "http://data.muziekschatten.nl/onderwerpen",
+  "@type": "Dataset",
+  "name": [
+    {
+      "@language": "nl",
+      "@value": "Muziekschatten - onderwerpen"
```

Comment:
For existing sources, we have `name colon subject`, e.g. `Muziek: genres en stijlen`. Could we standardise on that notation? This one would then become `Muziekschatten: onderwerpen`.

Reply:
Good suggestion, I'll change this.